views/events/calls_events.view.lkml

```diff
@@ -0,0 +1,217 @@
+# The name of this view in Looker is "Calls Events"
+view: calls_events {
+  # The sql_table_name parameter indicates the underlying database table
+  # to be used for all fields in this view.
+  sql_table_name: "EVENTS"."CALLS_EVENTS"
+    ;;
+  drill_fields: [id]
+  # This primary key is the unique key for this table in the underlying database.
+  # You need to define a primary key in a view in order to join to other views.
+
+  dimension: id {
+    primary_key: yes
+    type: string
+    sql: ${TABLE}."ID" ;;
+  }
+
+  # Here's what a typical dimension looks like in LookML.
+  # A dimension is a groupable field that can be used to filter query results.
```

Comment:
I would remove the first two rows.

Reply:
I'm mostly referring to comments like "Here's what a typical dimension looks like in LookML.", I assume they are canned comments coming from some template that can be removed and should add instead more useful/topical ones.